### PR TITLE
Folder Added : Assignment 5 and Assignment 6

### PR DESCRIPTION
**Pull request by pritikekan**

### CODE REVIEW

File: priti/HTML/Assign.6/Q1.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="Q1.css" />
+        <title><Assign6</title>
+    </head>
+    <body>
+        <h1 style="background-color: rgb(185, 211, 238); text-align: right;">MI-9 Intern HTML Assesment</h1>
```
Comment (pooja-nagare) [inline priti/HTML/Assign.6/Q1.html]: you are suppose to use external css not inline css

File: priti/HTML/Assign.6/Q1.css [hunk 2]
```diff
@@ -0,0 +1,12 @@
+body {
+    background-color: white;
+}
+h1 {
+    color: rgb(6, 6, 14);
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 200%;
+}
```
Comment (pooja-nagare) [inline priti/HTML/Assign.6/Q1.css]: use
border-bottom-left-radius: 8px;
border-top-right-radius: 8px;

**Review by Amit727619 — APPROVED** [review]
good

**Review by pooja-nagare — APPROVED** [review]

**Review by shivani-pawbake — APPROVED** [review]

**Review by jyoti-ing — CHANGES_REQUESTED** [review]
add space in css file

**Review by jyoti-ing — APPROVED** [review]

File: priti/HTML/Assign.5/Q1.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+            body {
+              background-image: url('123.jpeg');
+              background-repeat: no-repeat;
+              background-attachment: fixed;  
+              background-size: cover;
+              background-attachment: fixed; 
+            }
+            </style>
+    </head>
+    <h1 style="color:rgb(240, 21, 68);border: 2px solid black;">PRITI KEKAN</h1>
```
Comment (Pawan1005) [inline priti/HTML/Assign.5/Q1.html]: body tag missing

File: priti/HTML/Assign.6/Q1.html [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="Q1.css" />
+        <title><Assign6</title>
+    </head>
+    <body>
+     <div>
+        <div>
+        <h1  class="a">MI-11 Intern HTML Assesment</h1>
```
Comment (Pawan1005) [inline priti/HTML/Assign.6/Q1.html]: indentation missing

File: priti/HTML/Assign.6/Q1.html [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+        </div>
+         <hr/>
+        <div>
+        <h2 class="a"<i>personal Detail</i></h2>
```
Comment (Pawan1005) [inline priti/HTML/Assign.6/Q1.html]: indentation missing

File: priti/HTML/Assign.6/Q1.html [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+            <input type="radio" id="Age" name="age" value="less than 18" />
+            <label for="Age">less than 18</label>
+            <input type="radio" id="Age" name="age" value="" />
+            <label for="Age">18-25</label>
+            <input type="radio" id="Age" name="age" value="" />
+            <label for="Age">26-40</label>
+        </form>
+        <div>
+            <h2 class="a"><i>Acedemic Detail</i></h2>
+            </div>
```
Comment (Pawan1005) [inline priti/HTML/Assign.6/Q1.html:42]: indentation missing

File: priti/HTML/Assign.6/Q1.html [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+            <form>
```
Comment (Pawan1005) [inline priti/HTML/Assign.6/Q1.html]: indentation missing

File: priti/HTML/Assign.6/Q1.html [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+            <input type="checkbox" id="HTML" name=" SUB" value="" />
+            <label for="SUB">HTML</label>
+            <input type="checkbox" id="Javascript" name="SUB" value="" />
+            <label for="SUB"> Javasript</label>
+            <input type="checkbox" id="Java" name="SUB" value="" />
+            <label for="SUB"> Java</label>
+            <input type="checkbox" id="Oracle" name="SUB" value=">
+            <label for="SUB">Oracle</label>
+        </form>
+     <div>
```
Comment (Pawan1005) [inline priti/HTML/Assign.6/Q1.html]: stay focus 
where is closing of div tag

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

File: priti/HTML/Assign.5/Q1.html [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+              background-image: url('123.jpeg');
+              background-repeat: no-repeat;
+              background-attachment: fixed;  
+              background-size: cover;
+              background-attachment: fixed; 
+            }
+            </style>
+    </head>
+    <body>
+    <h1 style="color:rgb(240, 21, 68);border: 2px solid black;">PRITI KEKAN</h1>
```
Comment (Pawan1005) [inline priti/HTML/Assign.5/Q1.html]: indentation missing

File: priti/HTML/Assign.6/Q1.html [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+    </head>
+    <body>
+        <div>
+          <div>
+           <h1 class="a">MI-11 Intern HTML Assesment</h1>
+          </div>
+           <hr/>
+          <div>
+            <h2 class="a"<i>personal Detail</i></h2>
+          </div>
```
Comment (Pawan1005) [inline priti/HTML/Assign.6/Q1.html]: indentation missing

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**